docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md

```diff
@@ -93,6 +95,10 @@ The restore take a few seconds, after which expect to be logged out.
 
```

Comment:
Above "in most cases, you should choose the most recent backup in the vast majority of cases" redundant 'cases'

Reply:
fixed